File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void){

    int answer;
    answer = get_int("Height: ");
    while (answer > 8 || answer < 1){
        answer = get_int("Height: ");
    }

    int tags = 1;
    while (tags <= answer){
        for (int j = answer - tags; j > 0; j--){
            printf(" ");
        }
        for(int u = 0; u < tags; u++){
            printf("#");
        }
        tags++;
        printf("\n");
    }
}

File: mario-python/mario.py
from cs50 import get_int

answer = get_int("Height: ")
while answer < 1 or answer > 8:
    answer = get_int("Height: ")
    break

tags = 1
spaces = answer - 1;
while tags <= answer:
    for x in range(spaces):
        print(" ", end="")
        continue
    for u in range(tags):
        print("#", end="")
        continue
    print("\n", end="")
    tags += 1
    spaces -= 1